### PR TITLE
XTTS v1.1 GPT Trainer

### PR DESCRIPTION
**Pull request by Edresson**
XTTS GPT Trainer for XTTS v1.1.

## ToDos:

- [x] Update Docs
- [x] Update recipe with the released checkpoints
- [x] Rebase the PR to update the code with the changes done in XTTS v1.1 release.

### CODE REVIEW

**Review by erogol — APPROVED** [review]

**Comment by erogol** [comment]
Looks good to me. Waiting for @WeberJulian

**Comment by Yuzuuu69** [comment]
`[GitHubblog](url)`

**Review by Yuzuuu69 — CHANGES_REQUESTED** [review]
Duplicate of #

**Review by WeberJulian — APPROVED** [review]

**Comment by WeberJulian** [comment]
I'm not 100% sure that we should use those values BATCH_SIZE 3 and GRAD_ACUMM_STEPS 84 for fine-tuning. In some extreme cases (small datasets) a single step will be more than an epoch. Why not use the values we use internally 8 and 1 ?

**Comment by WeberJulian** [comment]
Otherwise, it looks good to me too. Great work. In future PRs, we can focus on reducing the VRAM footprint of fine-tuning to something more reasonable that you can do on collab with 16Gb of VRAM.

**Comment by markrmiller** [comment]
That's interesting that you use 8/1 internally. The comments say you need an effective batch size of like 230 or something for good quality. Regardless, it would seem difficult to use a small dataset, in my limited experience, the model overfits pretty quickly, even with a much reduced learning rate, even with a decent sized data set. I'm still trying to get anywhere near a good voice match before it overfits.
